[DOC IMP]Update folders and MD files.

silabs-junfan opened the pull request:
Update folders and MD files.

Review:
@ChengYuan-CY 
@MarkDing

ChengYuan-CY commented:
Below are the review comments on this pull request.
1.
The markdown file name should be connected with dash line "-" if multi words present.

2.
All of the images link should be "files/xxx" rather than "../files/xxx"

3.
you can refer to the following application note for detail descriptions:AN0002.1
Customer can refer to AN0002.1 and AN0002.2 for Wireless Gecko Series 1 and Wireless Gecko Series 2 products hardware design respectively.

4.
For additional information on the external oscillators, refer to the application note, AN0016.2
For additional information on the external oscillators, refer to the application note AN0016.1 or AN0016.2

silabs-junfan commented:
> Below are the review comments on this pull request.
> 1.
> The markdown file name should be connected with dash line "-" if multi words present.
Answer: Modified.
> 2.
> All of the images link should be "files/xxx" rather than "../files/xxx"
Answer: Modified.
> 3.
> you can refer to the following application note for detail descriptions:AN0002.1
> Customer can refer to AN0002.1 and AN0002.2 for Wireless Gecko Series 1 and Wireless Gecko Series 2 products hardware design respectively.
Answer: This document doesn't include series 2 chips, so it doesn't need to add AN0002.2.
> 4.
> For additional information on the external oscillators, refer to the application note, AN0016.2
> For additional information on the external oscillators, refer to the application note AN0016.1 or AN0016.2
Answer: This document doesn't include series 2 chips, so it doesn't need to add AN0016.2.

ChengYuan-CY commented:
The changes look good to me.
